codespell_lib/data/dictionary.txt

```diff
@@ -13133,6 +13204,7 @@ soluable->soluble
 solveable->solvable
 solwed->solved
 som->some
+some times->sometimes
```

Comment:
Codespell can't currently handle this. Please remove it and add it to the list in https://github.com/codespell-project/codespell/issues/74 .

Reply:
done